app/src/org/commcare/activities/connect/ConnectIDNetworkHelper.java

```diff
@@ -45,6 +45,8 @@ public PostResult(int responseCode, InputStream responseStream, IOException e) {
         }
     }
 
+    private static boolean isBusy = false;
```

Comment:
we should make `ConnectIDNetworkHelper` a singleton otherwise multiple threads can still do duplicate calls.

Reply:
Done